README.md

```diff
@@ -11,6 +11,11 @@ interfaces.
 of the MetalKube architecture as the method used to manage kubernetes
 applications.
 
+## Project Discussion
+
+* [MetalKube Development Mailing List](https://groups.google.com/forum/#!forum/metalkube-dev)
+* #cluster-api on Kubernetes Slack
```

Comment:
The message on slack about archiving the old group said to use sig-cluster-lifecycle. I'm not sure which is more appropriate. Do you want to list both?